src/NuGet.Clients/NuGet.VisualStudio.Client/registration.json

```diff
@@ -0,0 +1,100 @@
+{
+  "properties": {
+    "nuGetPackageManager.general.externalSettings": {
+      "type": "external",
+      "title": "@115;{5fcc8577-4feb-4d04-ad72-d6c629b083cc}",
+
+      // should be a resource ID in real registration
```

Comment:
This comment from my example could be removed, since this is now "real registration" :)